src/plugins/gauge/components/GaugeFormController.vue

```diff
@@ -100,13 +100,16 @@ export default {
     components: {
```

Comment:
The `GaugeFormController` is just straight up abusing the Forms API support for custom controls to shoehorn a totally customized form into the properties dialog for gauges. Out of scope for this PR, but I will file a followup to fix this, this is not at all how custom controls are supposed to be used.

Reply:
Can you point to the original intent of how custom controls were supposed to be used? Perhaps a github issue? Or elaborate here?